praw/models/reddit/live.py

```diff
@@ -8,5 +8,31 @@ class LiveThread(RedditBase):
 
     STR_FIELD = 'id'
 
+    def __eq__(self, other):
+        """Return whther the other instance equals the current."""
```

Comment:
whther -> whether

Might be worth noting here and in `__hash__` that this comparison is case sensitive whereas the super class's comparisons are not.